cypress/integration/shared.js

```diff
@@ -451,6 +451,12 @@ export const saveClusterDetails = (cy) => {
   // click the 'save' button in order to save changes in the cluster info
   cy.get('button[name="save"]', { timeout: VALIDATE_CHANGES_TIMEOUT }).should('be.enabled');
   cy.get('button[name="save"]').click();
+  cy.wait(2 * 1000);
```

Comment:
It would be better to block on an UI change instead of dummy waiting. But feel free to ignore this comment in case it would be too complicated in this case.

Just please be careful in case of potential subsequent checks, i.e. whether there are/arn't errors rendered - 2 seconds might not be enough and the test will miss errors ... Such thinking would be easier if you block processing on an UI change ^^^.

Reply:
As discusses only on error we have the error list, and there are no IDs to match on successful save.